docs/podman-pod-kill.1.md

```diff
@@ -0,0 +1,43 @@
+% podman-pod-kill "1"
+
+## NAME
+podman\-pod\-kill - Kills all containers in one or more pods with a signal
+
+## SYNOPSIS
+**podman pod kill** [*options*] *pod* ...
+
+## DESCRIPTION
+The main process of each container inside the pods specified will be sent SIGKILL, or any signal specified with option --signal.
+
+## OPTIONS
+**--all, -a**
+
+Sends signal to all containers associated with a pod.
+
+**--latest, -l**
```

Comment:
May have missed it, but do we have the notion of "latest" with pods too?  I know we do with containers.

Reply:
yup I added support last week!